package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "devDependencies": {
+    "@symfony/webpack-encore": "^0.7.1",
+    "node-sass": "^4.5.3",
+    "sass-loader": "^6.0.5"
+  },
+  "dependencies": {
+    "bloodhound-js": "^1.2.1",
+    "bootstrap-sass": "^3.3.7",
+    "bootstrap-tagsinput": "^0.7.1",
+    "bootswatch": "^3.3.7",
+    "eonasdan-bootstrap-datetimepicker": "^4.17.47",
+    "font-awesome": "^4.7.0",
+    "highlight.js": "^9.12.0",
+    "imports-loader": "^0.7.1",
+    "jquery": "^3.2.1",
+    "lato-font": "^3.0.0",
+    "moment": "^2.18.1",
+    "typeahead.js": "^0.11.1"
+  }
```

Comment:
@weaverryan it depends how you deploy actually. When deploying on heroku, the build happens on the heroku build server, and it does not include dev deps